crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -14,18 +14,10 @@ use crate::{
     primitives::{Aabb, Frustum},
 };
 
-/// User indication of whether an entity is visible
-#[derive(Component, Clone, Reflect, Debug)]
+/// User indication of whether an entity is visible as a marker component
+#[derive(Component, Copy, Clone, Reflect, Debug, Default)]
 #[reflect(Component)]
-pub struct Visibility {
-    pub is_visible: bool,
-}
-
-impl Default for Visibility {
-    fn default() -> Self {
-        Self { is_visible: true }
-    }
-}
+pub struct Visible;
```

Comment:
Once we have benchmarks for this, we should be testing how changing this component's storage type impacts performance.